policy/modules/system/systemd.te

```diff
@@ -1158,6 +1165,36 @@ seutil_read_file_contexts(systemd_sessions_t)
 
 systemd_log_parse_environment(systemd_sessions_t)
 
+
+#########################################
+#
+# Sysusers local policy
+#
+
+allow systemd_sysusers_t self:capability { chown fsetid };
+allow systemd_sysusers_t self:process setfscreate;
+allow systemd_sysusers_t self:unix_dgram_socket { create_socket_perms sendto };
+
+dev_write_kmsg(systemd_sysusers_t)
```

Comment:
Please see `systemd_log_parse_environment()` which will handle several of the perms here.